{"name": "test_register[wrong@mail.com-12345-400-error]", "status": "passed", "attachments": [{"name": "stdout", "source": "cf48ed0f-52a0-4c53-bf5c-cfa62b04b0c0-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "email", "value": "'wrong@mail.com'"}, {"name": "password", "value": "'12345'"}, {"name": "expected_status", "value": "400"}, {"name": "expected_key", "value": "'error'"}], "start": 1744226131381, "stop": 1744226131988, "uuid": "fdbe4f9d-ab4a-4a0d-a4fc-fc3dd1429373", "historyId": "e005a11a3140897832ad6a397d6e3753", "testCaseId": "4cdf5265c4a7bcfc67194c5978cf756a", "fullName": "tests.test_register#test_register", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_register"}, {"name": "host", "value": "CD4093DYG"}, {"name": "thread", "value": "1360-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_register"}]}